robot/demos/SkidSteerServos/SkidSteerServos.ino

```diff
@@ -0,0 +1,107 @@
+#include <ArduinoBlue.h>
+
+#include <SoftwareSerial.h>
+
+#include <Servo.h>
+
+Servo myservo;
```

Comment:
Like Peter said, it's good to have descriptive variable names for when the code gets more complex
```suggestion
Servo servoFrontRight;
Servo servoMiddleRight;
Servo servoBackRight;
Servo servoFrontLeft;
Servo servoMiddleLeft;
Servo servo BackLeft;
```